### PR TITLE
Solve the .303 problem

### PR DESCRIPTION
**Pull request by SerinaCE**
swapped the bolt action rifle to Ishapore 2A1 and the LMG to Bren L4, both in service for a decent while and look ~identical to what we had except for using .308, the Bren now has normal box mag but can load faster so it cancels out
adjusted recoil values, magazine size and burst count for lmg 

## References

https://www.wikiwand.com/en/Lee%E2%80%93Enfield#Ishapore_2A/2A1
https://www.wikiwand.com/en/Bren_light_machine_gun#L4

## Reasoning
having 2 standard full power rifle cartridges in basegame was always pointless, no reason to force player to switch standard cartridge midgame if we can just... not do that at 0 inconvenience.
also trader inventories.

re: reload time
Sheets doesn't have an option for *top feed box* magazines, there is a reason a whole bunch of interwar designed LMG had top feed, it's just much more convenient to load from prone. The option we do have was made specifically for the DP's pan mag (bulky, harder to position). 

## Alternative
Core: replace fake mag Bren with shroudless Lewis gun [(vanilla texture most probable based on)](https://discord.com/channels/278818534069501953/303988654492090370/1243709055026069625), (shroud replaced with hotswap barrel) couldn't find exact weight 10kg tops so that'll be fine.
Armory: motball Madsen, let me add Bren L4 and M968 Mosquefal with rifle grenades, **don't overrule reload time i set.**

### CODE REVIEW

**Comment by SamaelGray** [comment]
There's absolutely no need for something that was settled by the team recently to be changed for no reason, The Bren Mark IV looks close enough, the pan mag exists irl and it's being used in the single stack form and it weighs appropriate for something in its use-case.

The vanilla bolt-action is also a Lee-Enfield with .303 being the most common ammo used for the gun.

About the reload times, the sheet has the entry for Drum or Topfeed mags being 4.9 seconds, the shape of the mag does not matter, it's still topfed.

TL;DR, there is no need to fix something that's not broken based on subjective personal preferences.

**Comment by N7Huntsman** [comment]
The benefit of using .303 is that it was a widely-used cartridge that had a lot of platforms prior to the Cold War, but which dropped of sharply with the era of NATO standardization. This is _around_ the same period of time where weapons of the sort the more advanced research projects unlock become more common.

In that regard, the cost of transitioning from .303 to a more modern cartridge is intentional; there's a deliberate "step up" in technology, and the player has to decide whether to rush more advanced tech or invest more heavily in using .303 weapons.
